reference/release-notes/1.22.0.md

```diff
@@ -0,0 +1,92 @@
+
+### Streaming stack improvements
+
+* Streaming can now be enabled when launching an instance using the `--enable-streaming` option. Existing installations will require the charm deployment for Anbox Cloud to be manually upgraded to use the new relation between the AMS and the stream agent.
+
+  See [How to stream applications](https://anbox-cloud.io/docs/howto/application/stream) and [How to create an instance](https://anbox-cloud.io/docs/howto/instance/create) for details on launching stream-enabled instances. See [How to upgrade Anbox Cloud](https://anbox-cloud.io/docs/howto/update/upgrade-anbox) for instructions on manually upgrading the relation between the AMS and the stream agent charms.<!--Ac-2460 and AC-2360-->
+
+* VirGL is the default renderer for NVIDIA GPUs with driver version 545 and later. For driver versions earlier than 545 or old NVIDIA GPU architecture such as Turing, Ampere or Ada, EmuGL is used as the default renderer. See [Rendering architecture](https://anbox-cloud.io/docs/explanation/rendering-architecture).<!--AC-2459, AC-2270 and AC-2348-->
+* Experimental support for [pointer lock](https://developer.mozilla.org/en-US/docs/Web/API/Pointer_Lock_API) is available with this release. This helps in locking the pointer to a particular browser element, for example, your game window, and limits movement outside the game until the lock is disabled. The pointer lock property must be explicitly enabled on the stream object via the [streaming SDK](https://github.com/canonical/anbox-streaming-sdk).<!--AC-2379-->
+* All components of Anbox Cloud upgrade to NVIDIA 550 driver series by default on Ubuntu 22.04 images. For Ubuntu 20.04 images, 535 driver series will be used. <!--AC-2309-->
+
+### Dashboard improvements
+
+The following improvements are available in the Anbox Cloud dashboard for this release:
+
+* Page level guided tours are now available for help in navigating through the dashboard.
+* Support for using custom authentication server instead of Ubuntu SSO.
+* Custom instance name field in *Create instance* form.
+* User experience enhancements to the AAOS control panel:
+  - Extending VHAL support to allow setting the geographic location and to be able to provide and revoke consent for enabling APIs in the browser when using maps.
+* Enhancements to forms to support AOSP vs AAOS applications.
+
+### Other
+
+* Support for building custom images of Anbox Cloud based on AAOS. See [Custom images](tbd).<!--AC-2458-->
+* Android security updates for May 2024 (see [Android Security Bulletin - May 2024](https://source.android.com/docs/security/bulletin/2024-05-01) for more information).<!--AC-2437-->
+* The Android WebView has been updated to [124.0.6367.82](https://chromereleases.googleblog.com/2024/04/chrome-for-android-update_24.html).
+
+## Deprecations
+
+* Support for Ubuntu 20.04 (focal) will be dropped in a future release.
+* Support for the EmuGL renderer is removed in Anbox Cloud 1.23.0.
```

Comment:
I think we should be a tad more nuanced here. We are planning to remove these in the future, but this can change depending on various circumstances.
What about:
```suggestion
* Support for Ubuntu 20.04 (focal) is planned for removal in a future release.
* Support for the EmuGL renderer is planned for removal in Anbox Cloud 1.23.0.
```
Also, I think it would be good if we could tell in what release we plan to remove focal support.
IIRC we talked about removing support for focal in 1.23 or 1.24. cc @adglkh @morphis Do you recall what we had planned for this?

Reply:
@ajanon I'll leave Simon comment on this.